src/qml/qmlmixxxcontroller.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <QQuickItem>
+
+#include "controllers/rendering/controllerrenderingengine.h"
+
+namespace mixxx {
+namespace qml {
+
+class QmlMixxxController : public QQuickItem {
```

Comment:
can we rename this to `QmlMixxxControllerScreen`? Otherwise I wouldn't know what to call the hypothetical plain (screen-less) controller feature.